orthoinference/src/main/java/org/reactome/orthoinference/EventsInferrer.java

```diff
@@ -191,6 +191,7 @@ public static void inferEvents(Properties props, String pathToConfig, String spe
 				return;
 			} catch (Exception e) {
 				e.printStackTrace();
+				return;
```

Comment:
Since the catch block body for Exception and RuntimeException are the same now, they could be combined into one:
```
} catch (RuntimeException | Exception e) {
    e.printStackTrace();
    return;
}
```
The RuntimeException block could also just be removed if you don't mind not having it explicitly since RuntimeException is a child of Exception.  The Exception catch block will catch any kind of exception thrown.